client/packages/common/src/ui/components/popover/PaperPopover/PersistentPaperPopover/PersistentPaperPopover.tsx

```diff
@@ -0,0 +1,52 @@
+import React, { PropsWithChildren } from 'react';
+import Paper from '@mui/material/Paper';
+import { PaperPopoverProps, usePopover } from '../..';
+import { CloseIcon } from '@common/icons';
+import { Box, ClickAwayListener, IconButton } from '@mui/material';
+
+export function PersistentPaperPopover({
```

Comment:
Didn't want the popover to close when we added a new item... since the `usePaperClickPopover` closes as soon as you do an action.. will create a refactor issue 👀